docs/quickstart.md

```diff
@@ -1,70 +1,163 @@
+- Running trials using machine learning (ML) to determine the best configuration
 
-## Prerequisites
+If you encounter any issues or require assistance, please reach out to us via [slack][rso slack] or [contact us][contact us form].
 
-You must have a Kubernetes cluster. Additionally, you will need a local configured copy of `kubectl`. The Red Sky Ops Tool will use the same configuration as `kubectl` (usually `$HOME/.kube/config`) to connect to your cluster.
+## Prerequisites
 
-A local install of [Kustomize](https://github.com/kubernetes-sigs/kustomize/releases) (v3.1.0+) is required to build the objects for your cluster.
+- Kubernetes cluster
+  - A typical [minikube][minikube] cluster will be sufficient. The experiment may consume up to 4vcpu and 4GB memory
+- [kubectl][kubectl] properly configured for your cluster
+- [kustomize][kustomize] v3.1.0+
+- [redskyctl][install redskyctl]
+- [Red Sky Ops Account][rso account]
 
-If you are planning to create the postgres experiment from this guide, a [minikube](https://kubernetes.io/docs/setup/learning-environment/minikube/) cluster will be sufficient.
+## Initialize the Red Sky Ops Manager
 
-## Install the Red Sky Ops Tool
+After installing [redskyctl][install redskyctl] and creating an [account][rso account], log in to your Red Sky Ops account:
+```sh
+$ redskyctl login
+```
+You should see output similar to this:
+```sh
+Opening your default browser to visit:
 
-[Download](https://github.com/redskyops/redskyops-controller/releases) and install the `redskyctl` binary for your platform. You will need to rename the downloaded file and mark it as executable.
+	https://auth.carbonrelay.io/authorize?...
 
-For more details, see [the installation guide](install.md).
+You are now logged in.
+```
 
-## Initialize the Red Sky Ops Manager
+When you initialize your cluster for Red Sky Ops, the following resources will be created:
+- `redsky-system` namespace
+- `experiment` and `trial` custom resource definitions
+- `ClusterRole` and `ClusterRoleBinding` for the controller service account
+- `redsky-controller-manager` deployment
+- an authorization secret for the controller
 
-Once you have the Red Sky Ops Tool you can initialize the manager in your cluster:
+Initialize your cluster with the following command:
 
 ```sh
-$ redskyctl init
+$ redskyctl init --wait
+```
+You should see output similar to this:
+```sh
+namespace/redsky-system created
+customresourcedefinition.apiextensions.k8s.io/experiments.redskyops.dev created
+customresourcedefinition.apiextensions.k8s.io/trials.redskyops.dev created
+clusterrole.rbac.authorization.k8s.io/redsky-manager-role created
+clusterrolebinding.rbac.authorization.k8s.io/redsky-manager-rolebinding created
+deployment.apps/redsky-controller-manager created
+clusterrole.rbac.authorization.k8s.io/redsky-patching-role created
+clusterrolebinding.rbac.authorization.k8s.io/redsky-patching-rolebinding created
+secret/redsky-manager created
```

Comment:
This is too specific and too likely to change rendering the documentation out-of-date.

Reply:
I'll drop them as part of the doc. As a followup ( doesn't have to be crammed in this release ) we should try to summarize the output with a simple `success`.